src/main/java/InputReader.java

```diff
@@ -0,0 +1,48 @@
+/**
+ *
+ */
+public class InputReader {
```

Comment:
Good use of pascal casing for naming of classes! Appreciate the attention to detail :)